src/thunderbots/software/radio_communication/mrf/annunciator.cpp

```diff
@@ -19,6 +21,11 @@ namespace
      */
     const double REQUEST_BUILD_IDS_INTERVAL = 0.5;
 
+    /**
+     * Amount of time to keep an edge-triggered message sending, in seconds.
+     */
+    const int ET_MESSAGE_KEEPALIVE_TIME = 5;
```

Comment:
Thought on changing this to 10? Feel like it just gives a bit more time to walk to the computer and take note of what's there.

Reply:
Yep, 10 seems to be a better delay.